include/mata/nfa/algorithms.hh

```diff
@@ -68,6 +68,8 @@ Nfa complement_classical(const Nfa& aut, const Mata::Util::OrdVector<Symbol>& sy
  * i.e., if the final intersection of smaller complement of bigger is empty.
  */
 bool is_included_naive(const Nfa& smaller, const Nfa& bigger, const Alphabet* alphabet = nullptr, Run* cex = nullptr);
+//naive is not a good word, maybe textbook, or explicit, construct_first
+//the alphabet parameter is useless in inclusion
```

Comment:
Why do you need an alphabet in inclusion?

Reply:
Because inclusion is computed using a complement of an automaton, which in our interface requires the alphabet.

Reply:
In inlcusion, the complement can be with respect to the symbols used in the other automaton, not with respect to an external alphabet.

Reply:
As @Adda0 said (and it is clearly explained in the comment for this function), alphabet is there as an optimization if you already have it. Otherwise, you have to go trough the whole delta of both automata to create the alphabet, it is possible that could be costly. But maybe not, you are going to do complement anyway, that will probably take much more time.

Reply:
I think so, the efficient antichain inclusion does not need the alphabet at all, the explicit one will do much more constly operations than collecting alphabet anyway.